### DISCUSSION TITLE
Can we do a graduation ceremony for CNCF mentees?

### DISCUSSION THREAD

**Discussion by thisisobate** [cncf/mentoring#673] [General]
How about we do a graduation ceremony for mentees? something like the [CNCF Mentees Meetup?](https://community.cncf.io/cncf-mentees-meetup/)

As a past mentee myself, I figured this was one of the things that I looked forward to, and thus, my motivation for co-organizing the first CNCF mentees meetup.

What do y'all think?

**Comment by nate-double-u** [comment]
I think this is a fun idea--I agree with [Chris' comment](https://github.com/cncf/mentoring/issues/669#issuecomment-1191483550) about including a graduation ceremony as a part of a larger monthly meetup focused on mentorship.

@bwplotka may have some ideas too!
